### PR TITLE
fix relative schema urls

### PR DESCRIPTION
**Pull request by clemensv**
This PR fixes the relative relationships between the various schemas in the repo.

### CODE REVIEW

File: endpoint/schemas/xregistry_endpoint_protocol_options.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema",
+  "$schema": "http://json-schema.org/draft-07/schema#",
```
Comment (duglin) [inline endpoint/schemas/xregistry_endpoint_protocol_options.json:2]: I'm curious, why is the "#" at the end needed?

File: message/schemas/xregistry_openapi_messagedefinition.json [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+  "openapi": "3.1.0",
   "info": {
     "title": "xRegistry API",
     "description": "xRegistry API",
     "version": "0.5-wip"
   },
-  "servers": [],
+  "servers": [
+    {
+      "url": "https://raw.githubusercontent.com/clemensv/spec-1/fix-schemas/message/schemas/"
```
Comment (duglin) [inline message/schemas/xregistry_openapi_messagedefinition.json:10]: should this be pointing to an xreg URL?

Comment (clemensv) [reply message/schemas/xregistry_openapi_messagedefinition.json:10]: I will revert that bit.

File: endpoint/schemas/xregistry_endpoint_protocol_options.json (hunk 1 above)
Comment (clemensv) [reply endpoint/schemas/xregistry_endpoint_protocol_options.json:2]: Apparently that is the common form to refer to draft-07 as I found

File: message/schemas/xregistry_openapi_messagedefinition.json (hunk 2 above)
Comment (clemensv) [reply message/schemas/xregistry_openapi_messagedefinition.json]: https://github.com/swagger-api/swagger-editor/issues/4241

**Comment by duglin** [comment]
Approved on the 6/29 call

**Pull request merged**